<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainScreen</class>
 <widget class="QWidget" name="MainScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>783</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>무인 도서 관리 시스템</string>
  </property>
  <widget class="QLabel" name="title_label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>20</y>
     <width>400</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>30</pointsize>
    </font>
   </property>
   <property name="text">
    <string>무인 도서 관리 시스템</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="book_list_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>도서 목록</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="search_edit">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>331</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>도서명, 저자명, ISBN으로 검색</string>
   </property>
  </widget>
  <widget class="QPushButton" name="search_btn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QLabel" name="user_name_label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>50</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="text">
    <string>님, 안녕하세요.</string>
   </property>
  </widget>
  <widget class="QListWidget" name="book_list">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>501</width>
     <height>591</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="book_name_label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>170</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>책 이름 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="book_author_label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>210</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>저자 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="book_isbn_label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>250</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ISBN : </string>
   </property>
  </widget>
  <widget class="QLabel" name="book_rental_label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>290</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>대여 상태 : </string>
   </property>
  </widget>
  <widget class="QLabel" name="book_location_label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>330</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>위치 : </string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
